tasks/install_dist_pkgs.yml

```diff
@@ -51,8 +58,8 @@
     repo_gpgcheck: 'yes'
     state: present
     description: New Relic Infrastructure
-  when: nrinfragent_distribution == 'amazon'
-  register: setup_agent_repo
+  when: ansible_distribution|lower == 'amazon'
```

Comment:
@JakubKotkowiak is there a reason to not use `nrinfragent_distribution` here?

Reply:
It is being used just making sure that in case the distro isn't all lower case that it can still read it

Reply:
Ah my bad, I completely missed the point. Fixing it now

Reply:
Cool, thanks.